src/components/personkort/PersonkortChangeEnhet.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import { BodyLong, Button, Heading, Modal } from "@navikt/ds-react";
+import { BehandlendeEnhet } from "@/data/behandlendeenhet/types/BehandlendeEnhet";
+import styled from "styled-components";
+import { useChangeEnhet } from "@/components/personkort/useChangeEnhet";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import ModalWrapper from "nav-frontend-modal";
+
+const texts = {
+  endre: "Endre til",
+  innholdModal:
+    "Hvis du ikke har tilgang til NAV utland, vil du miste tilgangen til denne personen når enheten endres.\n" +
+    "Veiledere med tilgang til NAV utland kan senere flytte personen tilbake til geografisk enhet.",
+};
+
+const NAV_UTLAND = "0393";
+
+const ButtonGroup = styled.div`
+  > * {
+    margin-right: 1em;
+  }
+`;
+
+interface PersonkortChangeEnhetProps {
+  behandlendeEnhet: BehandlendeEnhet;
+}
+
+const PersonkortChangeEnhet = ({
+  behandlendeEnhet,
+}: PersonkortChangeEnhetProps) => {
+  const [open, setOpen] = useState(false);
+  const fnr = useValgtPersonident();
+  const changeEnhet = useChangeEnhet(fnr);
+
+  const isCurrentlyNavUtland = behandlendeEnhet.enhetId === NAV_UTLAND;
+  const heading = `${texts.endre} ${
+    isCurrentlyNavUtland ? "geografisk enhet" : "NAV utland"
+  }`;
+
+  const updateEnhet = () => {
+    changeEnhet.mutate({
+      personident: fnr,
+      isNavUtland: !isCurrentlyNavUtland,
+    });
+    setOpen(false);
+  };
+
+  return (
+    <>
+      <Button variant="secondary" onClick={() => setOpen(true)}>
+        {heading}
+      </Button>
+      <ModalWrapper
+        onRequestClose={() => setOpen(true)}
+        contentLabel={"a"}
+        isOpen={open}
+      >
```

Comment:
Måtte wrappe modalen med denne, se slack-tråd hos designsystemet: https://nav-it.slack.com/archives/C7NE7A8UF/p1665141121961199